pom.xml

```diff
@@ -57,15 +57,15 @@
   <properties>
     <revision>2.19</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.346.3</jenkins.version>
```

Comment:
I thing you shall keep original version here. It shall be updated by dependabot (when I understand it correct) And now it cause that the job fails. So revert changes in the pom file. (@MarkEWaite  I hope it is correct what I write here)

Reply:
If you intend to drop support for Java 8 completely, the way to do that is to make the `jenkins.version` 2.361.1 or later.  Those versions require Java 11 or newer.

However, that change will likely fail in a test from an unprivileged user, since the Jenkinsfile from the unprivileged user won't be executed.